src/components/Reports/ProjectMemberTable/ProjectMemberTable.jsx

```diff
@@ -1,4 +1,4 @@
-import { Stub } from '../../common/Stub';
+import Stub from '../../common/Stub';
 import React, { useEffect, useState } from 'react';
```

Comment:
```suggestion
import { useEffect, useState } from 'react';
```
Doesn't seem like `React` is being used